### PR TITLE
Separated Slack bot docs from README

### LINKED ISSUE #3: Move out slack documentation from README
README is very cluttered right now. Only because of slack documentation with images.

Suggestion:

Create another documentation file in `docs` folder and move out Slack integration steps from README. And only add the link for the new document.

### PR DESCRIPTION
**Pull request by neodark**
This helps removed the cluttered part of the README file by separating Slackbot documentation in docs/

Closes #3

### CODE REVIEW

File: docs/SLACK_BOT.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## How to setup the Slack Bot?
```
Comment (mukarramali) [inline docs/SLACK_BOT.md:1]: This is a separate doc, heading should have just one #

File: docs/SLACK_BOT.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+1. Create a [Slack App](https://api.slack.com/apps?new_app=1).
+2. Check your newly created app in [Dashboard](https://api.slack.com/apps/)
+3. Go back to Slack. Create a channel, give a name(ex: vacabot)
+4. In left side bar, choose *Incoming Webhooks* and *Activate Incoming Webhooks*.\
+   Scroll down to *Webhook URLs for Your Workspace* and add a new one.
+   ![Webhooks](./docs/vacabot-webhooks.png)
```
Comment (mukarramali) [inline docs/SLACK_BOT.md:8]: The relative paths for images are wrong.
`./vacabot-webhooks.png`
The same goes for other images.

**Review by mukarramali — CHANGES_REQUESTED** [review]

File: docs/SLACK_BOT.md (hunk 2 above)
Comment (neodark) [reply docs/SLACK_BOT.md]: Good catch. Just updated again

File: docs/SLACK_BOT.md (hunk 1 above)
Comment (neodark) [reply docs/SLACK_BOT.md:1]: Usage goes under it though. Is that correct?

Comment (neodark) [reply docs/SLACK_BOT.md:1]: Updated as requested.

**Comment by mukarramali** [comment]
@neodark LGTM

**Review by mukarramali — APPROVED** [review]

**Pull request merged**